docs/_docs/1_1_spark_quick_start_guide.md

```diff
@@ -75,6 +75,7 @@ df.write.format("hudi").
   option(PRECOMBINE_FIELD_OPT_KEY, "ts").
   option(RECORDKEY_FIELD_OPT_KEY, "uuid").
   option(PARTITIONPATH_FIELD_OPT_KEY, "partitionpath").
+  option(KEYGENERATOR_CLASS_PROP, "org.apache.hudi.keygen.SimpleKeyGenerator").
```

Comment:
Can we remove this entry. We want to keep the number of configs small in quick start. May be in the "Note" section, you can call out that, default value for key gen prop is SimpleKeyGen which is what is being used for quick start in general. But if someone is interested in non-partitioned key gen, please use NonpartitionedKeyGenerator. Also, you can link to the key gen blog for someone to refer to all diff key gens supported